whylog/log_reader/searchers.py

```diff
@@ -82,16 +82,20 @@ def _reverse_from_offset(self, offset, buf_size=BufsizeConsts.STANDARD_BUF_SIZE)
                     if len(line):
                         actual_offset = self._decrease_actual_offset_properly(actual_offset, line)
                         yield line, actual_offset
-            actual_offset = self._decrease_actual_offset_properly(actual_offset, truncated)
-            yield truncated, actual_offset
+            if truncated:
+                actual_offset = self._decrease_actual_offset_properly(actual_offset, truncated)
+                yield truncated, actual_offset
 
-    def search(self, investigation_step):
+    def search(self, investigation_step, original_front_input):
         clues = defaultdict(list)
-        offset = self._deduce_offset(investigation_step)
+        if original_front_input.line_source == self._file_path:
+            # TODO checking if host is also the same
+            offset = original_front_input.offset
```

Comment:
`min_delta` - do you mean TimeConstraints parameter? This problem hasn't been considered yet. This could be fixed by returning to the previous solution, but then I do not use the starting offset. I can create ticket to consider and fix this problem.